r&d.md

```diff
@@ -1134,7 +1138,7 @@
   ? Do we need extending root scope? Like writing some new props to it?
   + `with` doesn't allow writing new props anyways
 
-## [ ] Componentization: what can be done? -> likely no for now. When html-include is there we can talk
+## [x] Componentization: what can be done? -> likely no for now. When html-include is there we can talk
```

Comment:
have a look at ponys.js, i like how that creates web components from templates https://github.com/jhuddle/ponys

Reply:
Ponys is nice! Impressive - they made very simple DCE. I think it can play well with sprae actually, since it allows easy templates. Thanks, I will research